### PR TITLE
Updates from django-template May 2017

### PR DESCRIPTION
**Pull request by tomkins**
Also a few tweaks to keep things updated for DDC apps.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -29,4 +29,4 @@ Create a site using this template:
 
 .. code-block:: console
 
-    $ cookiecutter gh:blancltd/digitaldiocese-django-template
+    $ cookiecutter gh:developersociety/django-template
```
Comment (zatan) [inline README.rst]: is this should be `developersociety/digitaldiocese-django-template`?

Comment (tomkins) [inline README.rst]: Good catch

File: {{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/settings/base.py [hunk 2]
```diff
@@ -13,49 +11,35 @@
...
 # Email
 # https://docs.djangoproject.com/en/{{ cookiecutter.django_version }}/ref/settings/#email
-ADMINS = (
-    ('Blanc Ltd', 'studio@blanc.ltd.uk'),
-)
 
+ADMINS = (('Blanc Ltd', 'studio@blanc.ltd.uk'), )
 MANAGERS = ADMINS
 
 SERVER_EMAIL = '{{ cookiecutter.project_slug }}@blanctools.com'
```
Comment (zatan) [inline {{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/settings/base.py]: devsoc?

Comment (zatan) [inline {{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/settings/base.py]: Make sure change it acrross

**Pull request merged**